README.md

```diff
@@ -12,11 +12,11 @@ $ apm install linter-erlc
 ```
 
 ## Settings
-You can configure linter-elixirc by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
+You can configure linter-erlc by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
 
 ```
-'linter-elixirc':
```

Comment:
I would rather the Settings section be a guide to the settings, instead of something users can directly copy (and then wonder why it is broken). If you want an example of what I mean [this](https://github.com/AtomLinter/linter-erb#settings) was just recently re-written in the style I'm talking about.

If you really want to keep this current style please provide a configuration that will actually work when copied on some system, because users _will_ (for whatever reason) just copy it directly. As it stands this configuration will break for everyone since you have `null` in there.